src/Orchard/Environment/Extensions/Models/ExtensionDescriptor.cs

```diff
@@ -12,6 +12,8 @@ public class ExtensionDescriptor {
         /// </summary>
         public string Id { get; set; }
 
+        public string VirtualPath { get { return Location + "/" + Id; } }

```

Comment:
Is this really needed here? Also don't concatenate paths by hand, use VirtualPathUtility.Combine().